fusionfrontend/src/app/store/asset-type-template/asset-type-template.model.ts

```diff
@@ -26,9 +26,14 @@ export class AssetTypeTemplate extends BaseEntity {
   imageKey: string;
   assetTypeId: ID;
   assetType: AssetType;
-  published: boolean;
+  publicationState: PublicationState;
   publishedDate: Date;
   publishedVersion: bigint;
   fieldTargetIds: Array<ID>;
   fieldTargets: Array<FieldTarget>;
 }
+
+export enum PublicationState {
```

Comment:
enum also exists in backend. Version in backend is without value assignment. Is this difference on purpose?